crates/net/downloaders/src/headers/metrics.rs

```diff
@@ -0,0 +1,30 @@
+use metrics::Counter;
+use reth_interfaces::p2p::error::DownloadError;
+use reth_metrics_derive::Metrics;
+
+/// The header downloader metrics.
+#[derive(Metrics)]
+#[metrics(scope = "downloaders_headers")]
+pub struct HeaderDownloaderMetrics {
+    /// The number of headers that were successfully sent to the poller (stage)
+    pub(crate) total_flushed: Counter,
+    /// Number of headers that were successfully downloaded
+    pub(crate) total_downloaded: Counter,
```

Comment:
pedantic style nit:

I don't like mixing accessors here, I guess we can make all of them `pub(crate)`?